tpie/pipelining/node.h

```diff
@@ -103,6 +107,37 @@ class node {
 	///////////////////////////////////////////////////////////////////////////
 	virtual ~node() {}
 
+	inline int get_minimum_file_descriptors() const {
+		return m_parameters.minimumFileDescriptors;
+	}
+
+	inline int get_maximum_file_descriptors() const {
+		return m_parameters.maximumFileDescriptors;
+	}
+
+	///////////////////////////////////////////////////////////////////////////
+	/// \brief Get the amount of memory assigned to this node.
+	///////////////////////////////////////////////////////////////////////////
+	inline int get_available_file_descriptors() const {
+		return m_availableFileDescriptors;
+	}
+
+	///////////////////////////////////////////////////////////////////////////
+	/// \brief Get the amount of memory assigned to this node.
+	///////////////////////////////////////////////////////////////////////////
+	inline int get_used_file_descriptors() const {
+		int ans=0;
+		/*for (const auto & p: m_buckets)
+			if (p) ans += p->count;*/
+		return ans;
+	}
+
+	void set_file_descriptor_fraction(double f);
+
+	inline double get_file_descriptor_fraction() const {
+		return m_parameters.fileDescriptorFraction;
+	}
+
```

Comment:
I know, but I was just using the same style as the other functions which almost all use `inline`. Should I still change it?